dist/mk-docker-opts.sh

```diff
@@ -11,7 +11,7 @@ OPTIONS:
 	-c	Output combined Docker options into DOCKER_OPTS var
 	-k	Set the combined options key to this value (default DOCKER_OPTS=)
 	-m	Do not output --ip-masq (useful for older Docker version)
-" >/dev/stderr 
```

Comment:
What's the reason for this change? AFAICT this isn't related from the sh->bash switch